<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrowseTasksSmall</class>
 <widget class="QWidget" name="BrowseTasksSmall">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>49</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,4,4,0">
     <item>
      <widget class="QLabel" name="shotLabel">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Shot: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="browseTasksButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>Select Task</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="homeButton">
       <property name="minimumSize">
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="BrowseTasksSmall.qrc">
         <normaloff>:/home.png</normaloff>:/home.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="BrowseTasksSmall.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>browseTasksButton</sender>
   <signal>clicked()</signal>
   <receiver>BrowseTasksSmall</receiver>
   <slot>showHideTree()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>BrowseTasksSmall</receiver>
   <slot>closeTree()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>homeButton</sender>
   <signal>clicked()</signal>
   <receiver>BrowseTasksSmall</receiver>
   <slot>goHome()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showHideTree()</slot>
  <slot>closeTree()</slot>
  <slot>goHome()</slot>
 </slots>
</ui>
